pkg/orchestrator/pipelines_test.go

```diff
@@ -132,11 +137,12 @@ func TestPipelineOrchestrator_Update_Success(t *testing.T) {
 		Return(want, nil)
 
 	got, err := orc.Pipelines.Update(ctx, plBefore.ID, newConfig)
-	assert.Equal(t, got, want)
-	assert.Ok(t, err)
+	is.Equal(got, want)
+	is.NoErr(err)
```

Comment:
I think it's better to check for the error first, so that if it fails, it shows that there's an error, instead of saying that got!=want. WDYT?